apps/extension/src/App/Settings/ExtraSettings/DeleteAccount/DeleteAccount.components.ts

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+export const Input = styled.input`
```

Comment:
I think at some point (not in this PR) we could incorporate `Input`, `InputFeedback` and `InputContainer` into `@anoma/components` and standardize usage across the apps, but I think it would be good to first take a good look at our existing `Input` there and see if it suits our needs to be used everywhere. That's a future task that could be improved when we're incorporating designs I think